pkg/config/setup/config.go

```diff
@@ -1500,6 +1500,9 @@ func logsagent(config pkgconfigmodel.Setup) {
 
 	// Max size in MB to allow for integrations logs files
 	config.BindEnvAndSetDefault("logs_config.integrations_logs_files_max_size", 100)
+
+	// Add a tag to file logs that are truncated by the agent
+	config.BindEnvAndSetDefault("logs_config.tag_truncated_logs", false)
```

Comment:
Would it make sense to add it to the config template?

Reply:
I was planning to leave this undocumented as it's a bit of an edge case, and probably will only be useful in diagnostics situations. I can add it if you disagree though.